common/game_concepts/wc_magic_concepts.txt

```diff
@@ -0,0 +1,5 @@
+﻿wc_mana = {
```

Comment:
You should add some debug actions to spend the mana, so we can test your icons.

Reply:
`add_%magic%_magic_lifestyle_xp`

Reply:
> Did you meant progressbars?
Yes.

> `add_%magic%_magic_lifestyle_xp`
So there only is a way to increase max mana?

Reply:
Use `explorer` console command then

Reply:
> Use `explorer` console command to edit character variables?

But why you don't create debug action? It's faster to click than console commands.

Reply:
now `wild_goose_chase_decision` sets your mana to zero